config/locales/activerecord.yml

```diff
@@ -1,5 +1,10 @@
+dictionary:
+  job_title_blank: &job_title_blank Enter a job title
+  job_description_blank: &job_description_blank Enter a job description
+  minimum_salary_blank: &minimum_salary_blank Enter a minimum salary
 en:
   errors:
+    # format: "%{message}"
```

Comment:
Not sure if you meant to leave this in

Reply:
Yep. What to show what @C-gyorfi and I were thinking of implementing. 
Didn't want any other tests (especially unrelated to this PR) to fail so left this commented out.

Reply:
Got it 😄